tox.ini

```diff
@@ -1,15 +1,39 @@
-# content of: tox.ini , put in same dir as setup.py
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
We've looked into it and actually haven't found anything quite like what you're suggesting.
I've merged this for now, we'll revisit this later on (or possibly just write a very basic version of such a tool ourselves).